lib/src/transformer/impl_generation.dart

```diff
@@ -588,6 +593,66 @@ class ImplGenerator {
         staticVariablesImpl
     );
   }
+
+  /// Apply a workaround for an issue where, in the DDC, abstract getter or setter overrides declared in a class clobber
+  /// the inherited concrete implementations. <https://github.com/dart-lang/sdk/issues/29914>
+  ///
+  /// Fixes the issue by generating corresponding abstract getters/setters to complete the pair
+  /// for accessors with the `@override` annotation.
+  void fixDdcAbstractAccessors(
+    AccessorType accessorType,
+    NodeWithMeta<ClassDeclaration, annotations.TypedMap> typedMap,
+  ) {
+    var candidateAccessors = new List<MethodDeclaration>.from(
+        typedMap.node.members.where((member) =>
+            member is MethodDeclaration &&
+            (member.isGetter || member.isSetter) &&
+            !member.isSynthetic &&
+            !member.isStatic &&
+            member.metadata.any((meta) => meta.name.name == 'override')
```

Comment:
I think that's fine, seeing as the `@override` restriction is documented. I put that restriction in place to avoid patching accessors that might not need it.